### ISSUE TITLE
False Positive on collections.OrderedDict.move_to_end (raising E1101)

### ISSUE THREAD

**Issue by pwamej** [PyCQA/pylint#1872]
### Steps to reproduce
run pylint on
```python
from collections import OrderedDict
cache = OrderedDict()
cache['a'] = 1
cache.move_to_end('a')
```

### Current behavior
`E1101:Instance of 'OrderedDict' has no 'move_to_end' member`

### Expected behavior
No error

### pylint --version output
```
pylint3 1.8.2,
astroid 1.6.1
Python 3.6.3 (default, Oct  3 2017, 21:45:48)
[GCC 7.2.0]
```

This method was added in 3.2. Does pylint use some older Python version as a reference?

**Comment by brycepg** [comment]
Thanks for the report. The brain for `collections` just needs to be updated. We sometimes have to manually rewrite common classes so they can be properly understood by the linter.

**Comment by scontini76** [comment]
Same problem here using the code from @pwamej 

## pylint --version output

```
pylint 1.8.4,
astroid 1.6.3
Python 3.6.5 (default, Mar 30 2018, 06:41:53)
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```
